### PR TITLE
fix: updating bag to not remove nfts on click

### PR DESCRIPTION
**Pull request by aballerr**
https://uniswaplabs.atlassian.net/browse/WEB-2172

### CODE REVIEW

File: src/nft/components/bag/BagHeader.tsx [hunk 1]
```diff
@@ -10,6 +11,10 @@ const ClearButton = styled(ButtonText)`
   font-weight: 600;
   font-size: 14px;
   line-height: 16px;
+
+  :active {
+    text-decoration: none;
```
Comment (aballerr) [inline src/nft/components/bag/BagHeader.tsx:16]: text get's underline when clicked, it should not

File: src/nft/components/bag/BagHeader.tsx [hunk 2]
```diff
@@ -56,12 +63,22 @@ interface BagHeaderProps {
 }
 
 export const BagHeader = ({ numberOfAssets, closeBag, resetFlow, isProfilePage }: BagHeaderProps) => {
+  const sizing = useMemo(() => {
+    let power = 0
+
+    while (10 ** power <= numberOfAssets) {
+      power++
```
[the remark this answers is not in the document]
Comment (vm) [reply src/nft/components/bag/BagHeader.tsx:70]: hm. can you make a separate function for the sizing piece atleast? you can still memoize the result. also variables for the 14, 6 would be good.

[the remark this answers is not in the document]
Comment (aballerr) [reply src/nft/components/bag/BagHeader.tsx:70]: okay sure, by verbiage you want variables?

[the remark this answers is not in the document]
Comment (vm) [reply src/nft/components/bag/BagHeader.tsx:70]: just to explain why those specific numbers. you can do comments instead for variables are better in that situation

[the remark this answers is not in the document]
Comment (vm) [reply src/nft/components/bag/BagHeader.tsx:70]: this function is a bit hard to follow for someone without context

File: src/nft/components/bag/BagHeader.tsx [hunk 3]
```diff
@@ -62,16 +62,24 @@ interface BagHeaderProps {
   isProfilePage: boolean
 }
 
-export const BagHeader = ({ numberOfAssets, closeBag, resetFlow, isProfilePage }: BagHeaderProps) => {
-  const sizing = useMemo(() => {
-    let power = 0
+const BASE_SIZING = 14
+const INCREMENTAL_SIZING = 6
+
+const getCircleSizing = (numberOfAssets: number): string => {
```
Comment (vm) [inline src/nft/components/bag/BagHeader.tsx:68]: can you just do `numberOfAssets.toString().length`?

File: src/nft/components/bag/BagHeader.tsx [hunk 4]
```diff
@@ -55,13 +62,26 @@ interface BagHeaderProps {
   isProfilePage: boolean
 }
 
+const BASE_SIZING = 14
+const INCREMENTAL_SIZING = 6
+
+const getCircleSizing = (numberOfAssets: number): string => {
+  const numberOfCharacters = numberOfAssets.toString().length
+
+  // each digit adds 6px worth of width (approximately), so I set the height and width to be 6px larger for each digit added
```
Comment (vm) [inline src/nft/components/bag/BagHeader.tsx:71]: nit: don't think you need this comment anymore, the variables are very clear

**Review by vm — APPROVED** [review]
